Documentation for release (#337)
codeplaysoftware/cutlass-sycl: CHANGELOG-SYCL.md

# SYCL CUTLASS Changelog

> ## [RELEASE NAME GOES HERE](https://github.com/codeplay/cutlass-fork/releases/tag/v???) (2025-04-30)

Reviewer: ```suggestion
## [Cutlass 3.8 sycl backend version 0.1](https://github.com/codeplay/cutlass-fork/releases/tag/v???) (2025-04-30)
```

> 

Reviewer: ```suggestion
- Support for Intel GPU data Centre Max (1100  and 1550) 
- Support for Intel Arc B580 Battlemage 
```

Reviewer 2: We haven't tested B570. But it only differs in number of Xe cores and memory. I think it should be reasonable safe to assume that it'll work too.

- GEMM/StreamK/SplitK with support for bfloat16 data type

> - Flash attention prefill and decode with KV cache with support for bfloat16 data type

Reviewer: ```suggestion
- Flash attention prefill and decode with KV cache with be supported for bfloat16 data type
```

# SYCL CUTLASS Changelog

> ## [Cutlass 3.8 sycl backend version 0.1](https://github.com/codeplay/cutlass-fork/releases/tag/v???) (2025-04-30)

Reviewer: ```suggestion
## [Cutlass 3.8 sycl backend version 0.1](https://github.com/codeplay/cutlass-fork/releases/tag/v3.8~0.1) (2025-04-30)
```

# SYCL CUTLASS Changelog

> ## [RELEASE NAME GOES HERE](https://github.com/codeplay/cutlass-fork/releases/tag/v???) (2025-04-30)

Reviewer 3: @mehdi-goli @Ruyk to provide the format for the release version. 

The list of changes in this release include: 

- GEMM/StreamK/SliptK with support for bfloat16 data type.
- Support for epilogue operations with element wise bias
- Flash attention prefill and decode with KV cache. The data type supported is bfloat16
